doc/tut/group.xml

```diff
@@ -1211,6 +1211,12 @@ Reconstructing the
 automorphism group may of course result in the  loss of other information
 &GAP; had already gathered, besides the (not-so-)nice monomorphism.
 <P/>
+Prescribing a known map as the nice monomorphism of a group is possible
+only if this map knows that it is injective.
+In the above example, this is the case because the map is created as the
+composition of two maps (a nice monomorphism and an isomorphism)
+which know that they are injective.
```

Comment:
Perhaps this and/or the other new text could mention that one can ensure this is the case by either invoking `IsInjective(map)` (safe but potentially slow) or `SetIsInjective(map, true)` (fast but the caller has to make sure this is correct) ?